server/src/main/java/org/apache/cassandra/sidecar/routes/cdc/ListCdcDirHandler.java

```diff
@@ -94,6 +93,14 @@ protected void handleInternal(RoutingContext context,
                                   Void request)
     {
         String cdcDir = metadataFetcher.instance(host).cdcDir();
+        if (isNullOrEmpty(cdcDir))
+        {
+            context.response()
+                   .setStatusCode(HttpResponseStatus.SERVICE_UNAVAILABLE.code())
+                   .setStatusMessage("CDC not turned on for cluster")
+                   .end();
+            return;
+        }
```

Comment:
I expressed the concern a few days ago at https://github.com/apache/cassandra-sidecar/pull/163#discussion_r1936764719. And it now became true :D

So, we should not rely `null` value of `cdcDir` to decide cdc is not enabled. The `null` value only means the `cdcDir` is not configured. Server should only reply with "CDC directory is not configured in Sidecar".